examples/clock.py

```diff
@@ -0,0 +1,69 @@
+import asyncio
+
+from yapapi.payload import vm
+from yapapi.services import Service
+
+from service_manager import ServiceManager
+
+
+class ProviderClock(Service):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.command_executed_cnt = 0
+
+    @classmethod
+    async def get_payload(cls):
+        #   We use a "standard" yapapi blender image (because every provider has it downloaded)
+        #   Could be any other working image, the only requirement is /bin/date command
+        image_hash = "9a3b5d67b0b27746283cb5f287c13eab1beaa12d92a9f536b747c7ae"
```

Comment:
I tried this, and installation of `pyzmq` fails:
```
Building wheels for collected packages: pyzmq
  Building wheel for pyzmq (PEP 517): started
  Building wheel for pyzmq (PEP 517): finished with status 'error'
  ERROR: Command errored out with exit status 1:
   command: /usr/local/bin/python3 /usr/local/lib/python3.8/site-packages/pip/_vendor/pep517/in_process/_in_process.py build_wheel /tmp/tmp1d2bsoyg
       cwd: /tmp/pip-install-p8n1nrsf/pyzmq_4025c7c1f6ce4d5f90b33624cd617078
```

There are some details & solutions here:
https://stackoverflow.com/a/51916006

I think it would be better for the example use case to keep the Dockerfile as simple as possible so I'm against any complex solution.

Reply:
But why do you install `pyzmq` in the first place? The comment says:
> Could be any other working image, the only requirement is /bin/date command

Reply:
OK, sure. I thought this was a comment to the `python-shell` example :/

Reply:
5713154